components/clp-package-utils/clp_package_utils/scripts/native/decompress.py

```diff
@@ -299,7 +299,7 @@ def main(argv):
     json_extraction_parser = command_args_parser.add_parser(EXTRACT_JSON_CMD)
     json_extraction_parser.add_argument("archive_id", type=str, help="Archive ID")
     json_extraction_parser.add_argument(
-        "--target-chunk-size", type=int, help="Target chunk size.", required=True
+        "--target-chunk-size", type=int, help="Target chunk size (B)."
```

Comment:
I am fine with removing the default value, but can you make sure the script still works without specifying target-chunk-size?

Reply:
Yep, already tested on my end and it works.